<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>  <!-- Уменьшена ширина окна -->
    <height>600</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">
    /* ГЛОБАЛЬНЫЕ НАСТРОЙКИ */
    * {
        font-family: 'Nunito Sans', 'Segoe UI', sans-serif;
        font-size: 12pt;
        color: #5A5A5A;
    }

    /* ОСНОВНОЙ ФОН */
    QMainWindow, QDialog {
        background-color: #FAF9F5;
        padding: 15px;
    }

    /* КАРТОЧКИ ЭЛЕМЕНТОВ */
    QFrame, QGroupBox, QListWidget, QTableWidget {
        background-color: white;
        border-radius: 12px;
        border: none;
        padding: 12px;
        margin: 6px;
        box-shadow: 0 2px 8px rgba(0,0,0,0.05);
    }

    /* УНИФИЦИРОВАННЫЕ КНОПКИ */
    QPushButton {
        background-color: #D4E8F2;
        color: #4A4A4A;
        border-radius: 14px;
        padding: 10px 20px;
        border: none;
        font-weight: 600;
        min-width: 100px;
        transition: all 0.2s;
        font-size: 11pt;
    }

    /* ПОЛЯ ВВОДА - КОМПАКТНЫЕ */
    QLineEdit, QComboBox, QDateEdit {
        background-color: white;
        border: 1px solid #E0E0E0;
        border-radius: 10px;
        padding: 8px 12px;
        selection-background-color: #D4E8F2;
        font-size: 11pt;
        min-height: 36px;
    }

    /* ОСОБО УЗКОЕ ПОЛЕ ЦЕНЫ */
    #priceEdit {
        max-width: 80px;
    }

    /* ШИРОКОЕ ПОЛЕ КАТЕГОРИИ */
    #categoryCombo {
        min-width: 150px;
    }
   </string>

  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Название:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Категория:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="categoryCombo">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Цена:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="priceEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="productList"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="calculateButton">
        <property name="text">
         <string>Рассчитать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="countLabel">
        <property name="text">
         <string>Товаров: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totalLabel">
        <property name="text">
         <string>Итого: 0 руб.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
